src/features/Main/sections/Testimonial/index.tsx

```diff
@@ -0,0 +1,72 @@
+import { FC } from 'react';
+import {
+  Carousel,
+  CarouselContent,
+  CarouselItem,
+  CarouselNext,
+  CarouselPrevious
+} from '@/components/ui/carousel';
+import { Card } from '@/components/ui/card';
+import Image from 'next/image';
+import { getAssetUrl } from '@/lib/utils';
+import { testimoni } from '@/constants/testimoni';
```

Comment:
for external function calls are sorted by import package first, followed by the file

Reply:
is the order like this?
i remove import getAssetUrl cause i don't use it anymore in this program

![Screenshot (1405)](https://github.com/hammer-code/lms-fe/assets/120875756/cb9210ef-9b4d-4035-ae96-517d0979a343)

Reply:
yaps nice